developer_tools/malloc_dbg.c

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file
+ * @brief This file contains malloc_dbg, calloc_dbg, free_dbg and printLeaks implementations.
+ * @author tinouduart33
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include "malloc_dbg.h"

```

Comment:
Add a one-line comment explaining what is the library/header for (see the example below).
```c++
#include <iostream>   /// for IO operations
#include <vector>     /// for std::vector
```